protocol_driver_grpc_async_callback.cc

```diff
@@ -316,4 +323,167 @@ void ProtocolDriverGrpcAsyncCallback::ShutdownServer() {
   }
 }
 
+using grpc::Status;
+
+class CallData {
+ public:
+  CallData(Traffic::AsyncService* service, grpc::ServerCompletionQueue* cq,
+           std::function<std::function<void()>(ServerRpcState* state)>* handler,
+           DistbenchThreadpool* thread_pool)
+      : service_(service), cq_(cq),
```

Comment:
The file is not formatted correctly (here and multiple places).
Run "make clang-format" to fix all the issues (and push changes).